README.md

```diff
@@ -27,33 +27,44 @@ GRChombo is licensed under the BSD 3-Clause License. Please see LICENSE for deta
 
 ## Fawcett Tutorial
 
-First we need to get a copy of both the Chombo and GRChombo repositories.
+The following tutorial has been created to be aimed at Masters/early PhD
+students with some HPC experience. We also assume that you have an account set
+up on fawcett, and that you are happy with using GitHub. Inevitably it will not be 
+possible to cover everything, but if anything is unclear please do get in contact.
+
+To run simulations with GRChombo, first we need to get a copy of both the
+Chombo and GRChombo repositories.
 
 ### Getting Chombo and GRChombo
 
-Our copy of the Chombo repository can be found
-[here](https://github.com/GRChombo/Chombo). To get a copy, clone it using a
-command such as
+The GRChombo copy of the Chombo repository can be found
+[here](https://github.com/GRChombo/Chombo). To get a copy on your local
+machine, clone it using a command such as
 ```bash
 git clone https://github.com/GRChombo/Chombo.git ~/Chombo
 ```
+e.g. to clone into your home directory.
 
-Next clone the GRChombo repository (and checkout this branch) using a command
+Next, clone the GRChombo repository (and checkout this branch) using a command
 such as
 ```bash
 git clone -b training/fawcett https://github.com/GRChombo/GRChombo.git ~/GRChombo
 ```
+[Note that if you already have a copy of GRChombo on your local machine, you do not
+need to clone again and can just checkout the training/fawcett branch.]
```

Comment:
Likewise, I think we should remove this as this tutorial is specific to running on fawcett.
```suggestion
```

Reply:
How about
```suggestion
[Note that if you already have a copy of GRChombo on fawcett, you do not
need to clone again and can just checkout the training/fawcett branch.]
```

Reply:
For simplicity, I would rather just leave it out because then we need to ensure everyone's clones are up-to-date and if they have cloned to a different directory, they will have to modify all of the commands below.

Reply:
Ok sure